src/Helpers/LoopHelper.php

```diff
@@ -47,8 +48,8 @@ public static function addTimeoutToLoop(
   ): void {
     $loop->addTimer($minutes * 60, function () use ($loop, $minutes, $spinner, $output) {
       self::finishSpinner($spinner);
-      $output->writeln("Timed out after $minutes minutes!");
       $loop->stop();
+      throw new AcquiaCliException("Timed out after $minutes minutes!");
```

Comment:
This would skip printing the output URL right?

Reply:
Confirmed my own question:
https://github.com/acquia/cli/pull/350/files/ff4887ad59953beac33f64417417273226f214f0#diff-dc099926d287b14cf26be796bb6b26f83b5c8541ff2b9f78dbb358871f1ecdf9R107

Reply:
I'm on the fence about doing this. Sometimes the timeout happens but URLs are actually accessible a few seconds/minutes after. I'd rather have a better message that explains it's not us, it's the DNS system, and the 2 URLs will shortly be available.

Reply:
Its a failure nonetheless, and I think we should also consider removing the just created IDE if it expired DNS.

Reply:
I'v changed this to catch the exception and print an error followed by the IDE links.